data/schema.js

```diff
@@ -1,4 +1,4 @@
-var getbabelRelayPlugin = require('babel-relay-plugin');
-var schema = require('./schema.json');
+var getbabelRelayPlugin = require('babel-relay-plugin')
+var schema = require('./schema.json')
 
-module.exports = getbabelRelayPlugin(schema.data);
\ No newline at end of file
+module.exports = getbabelRelayPlugin(schema.data, { substituteVariables: true })
```

Comment:
https://github.com/facebook/relay/commit/a57a7c86839f1d35f74e4329320ccf98ef49cf1c

Is not needed! Thanks for catching.